src/lib/modules/client/terminal/LaunchSheet.svelte

```
@@ -125,6 +125,7 @@
   }
 
   function handleKeydown(event: KeyboardEvent): void {
+    if (!open) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)